synapseclient/api/json_schema_services.py

```diff
@@ -0,0 +1,217 @@
+import json
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    AsyncGenerator,
+    Dict,
+    Generator,
+    List,
+    Optional,
+    Union,
+)
+
+from synapseclient.api.api_client import rest_post_paginated_async
+
+if TYPE_CHECKING:
+    from synapseclient import Synapse
+
+
+async def bind_json_schema_to_entity(
+    synapse_id: str,
+    json_schema_uri: str,
+    *,
+    enable_derived_annos: bool = False,
+    synapse_client: Optional["Synapse"] = None,
+) -> Union[Dict[str, Any], str]:
+    """
+    <https://rest-docs.synapse.org/rest/PUT/entity/id/schema/binding.html>
+
+    Bind a JSON schema to an entity
+
+    Arguments:
+        synapse_id:      Synapse Entity or Synapse Id
+        json_schema_uri: JSON schema URI
+        enable_derived_annos:  If True, derived annotations will be enabled for this entity
```

Comment:
Should annotations be spelled out for this kwarg instead of annos? I don't think we are using much, if any, short hand in the new models.

Reply:
Okay got it. I can make that update.